### PR TITLE
Date of Birth validation & fix possible appointment bug

### PR DESCRIPTION
**Pull request by Hugo-Chia**

### CODE REVIEW

**Comment by Hugo-Chia** [comment]
This PR fixes #108 and #111

File: src/test/java/seedu/address/model/person/DateOfBirthTest.java [hunk 1]
```diff
@@ -27,6 +27,10 @@ public void isValidDateOfBirth() {
         assertFalse(DateOfBirth.isValidDate("")); // empty string
         assertFalse(DateOfBirth.isValidDate(" ")); // spaces only
         assertFalse(DateOfBirth.isValidDate("01/13/1991")); // month is over 12
+        assertFalse(DateOfBirth.isValidDate("01/01/1880")); // date is before 1990
+        assertFalse(DateOfBirth.isValidDate("31/12/1889")); // boundary condition, date is before 1900
```
Comment (AerollingOverDeepBlueSeas) [inline src/test/java/seedu/address/model/person/DateOfBirthTest.java:31]: LGTM! I like the inclusion of the test with boundary condition.
